### ISSUE TITLE
Install fails with error "The drive or UNC share you selected does not exist or is not accessible."

### ISSUE THREAD

**Issue by ggalloway** [OpenShot/openshot-qt#5124]
**Describe the bug:**
Openshot v2.4.4 was installed on D drive. Working fine, and great app BTW thanks. D drive was corrupted somehow and was removed from machine. Tried to reinstall from the latest nightly build and got the error "The drive or UNC share you selected does not exist or is not accessible." Tried to reinstall from the 2.4.4 installer and got the same error

**Steps to reproduce the behavior:**
1. Install Openshot on some non-system drive.
2. Remove the drive.
3. Try to reinstall.
4. Get error "The drive or UNC share you selected does not exist or is not accessible."

**Expected behavior:**
For the install to succeed

**System Details:**
- OpenShot Version v2.4.4 and v3.0 {{ log_context.openshot_qt_version }}
- Operating System: Windows 10 Pro 22H2

**Log Files:**
I deleted the log files per the suggestion on the "submit a bug" wizard on OpenShot website. The attempted install did not create any new log files.

**Exception / Stacktrace:**
![OpenShotInstallErr](https://user-images.githubusercontent.com/8963676/216043533-6ea474ab-a38e-460f-8f92-6fe7c6d98daa.JPG)

**Comment by Colorjet3** [comment]
I see the issue here.  Since Openshot was not gracefully uninstalled there is still remains a Registry entry indicating where 2.4.4 or any other version is installed.  So, when you try to install it again, it looks at the Registry, finds the entry for Openshot, and then sees that it was installed on the D: drive then it skips asking you where to install it and defaults to the D: drive.

You need to fix the Windows Registry.  Please be very careful not to accidentally change or delete other entries as it can cause other issues
1.  From the Windows Search box type Regedit.
2.  From the "Registry Editor" window navigate to:  HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Uninstall
3.  Click once on Uninstall key so it is selected.
4.  Press CTRL+F and in the "find what" field type "openshot" excluding the quotes.
5.  Click on Find Next.
6.  The Uninstall key will expand and in the right pane it will highlight the first instance of "Openshot" it found.
7.  In the left pane under the Uninstall container it will highlight a key that looks something like this:  {4BB0DCDC-BC24-49EC-8937-72956C33A470}_is1.
8.  Delete this entire key.
9.  Close Regedit.
10. Install 2.4.4.

This should take care of it.

**Comment by ggalloway** [comment]
Yep that fixed it, thanks. I figured it was a registry entry but when I searched in regedit for "openshot" I only looked at registry keys not values or data. That'll teach me.

BTW the behavior you described (i.e. the install looking for previous file path) is definitely appreciated and expected behavior from any app. Probably just need to check the path exists before trying to access it. Anyway thanks!
